lib/changelog_web/templates/news_item/meta/_featuring.html.eex

```diff
@@ -0,0 +1,7 @@
+<span class="news_item-toolbar-meta-item">
+<%= for person <- Episode.participants(@episode) do %>
+  <%= link(to: PersonView.external_url(person), title: person.name) do %>
+    <%= maybe_lazy_image(@conn, PersonView.avatar_url(person), "Featuring #{person.name}", style: "height: 25px; width: 25px; display: inline-block; margin: 0; padding: 0;") %>
```

Comment:
@codyjames I hardcoded some styles here to make it look _decent_ for prototype purposes. Please purge these during your design process.